website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -73,7 +73,7 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
 | `data.attributes.assessments-enabled`         | boolean | `false`   | (previously `drift-detection`) Whether or not Terraform Cloud performs health assessments for the workspace. May be overriden by the organization setting `assessments-enforced`. Only available for Plus tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode).                                                                                              |
 | `data.attributes.auto-apply`                  | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful, [with some exceptions](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply).                                                                                                                                                                                                                                                                                                                                                |
 | `data.attributes.description`                 | string  | (nothing) | A description for the workspace.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
-| `data.attributes.execution-mode`              | string  | `remote`  | Which [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) to use. Valid values are `remote`, `local`, and `agent`. When set to `local`, the workspace will be used for state storage only. This value must not be specified if `operations` is specified.                                                                                                                                                                                                                                          |
+| `data.attributes.execution-mode`              | string  | `remote`  | Which [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) to use. Valid values are `default`, `remote`, `local`, and `agent`. When set to `local`, the workspace will be used for state storage only. This value must not be specified if `operations` is specified.                                                                                                                                                                                                                                          |
```

Comment:
You can't actually pass `default` directly to the API. I think that we want to leave the valid values the same, and change the `Default` column from `remote` to `(nothing)`. Then we can expand the description to say that this attribute is specifying the workspace level overwrite for the value, and if it is omitted the organization default will be used instead.